README.md

```diff
@@ -125,6 +196,7 @@ Returns the newly created instance.
         }
     }
 
+    dizzy.register("val", new P
```

Comment:
Feel like something is missing here.  Should it be.

    dizzy.register("val", new P());

Or something like that?